src/graphql/account.rs

```diff
@@ -286,9 +286,24 @@ impl AccountMutation {
     ) -> Result<AuthPayload> {
         let store = crate::graphql::get_store(ctx).await?;
         let account_map = store.account_map();
+        let client_ip = get_client_ip(ctx);
 
         if let Some(mut account) = account_map.get(&username)? {
             if account.verify_password(&password) {
+                if let Some(allow_access_from) = account.allow_access_from.as_ref() {
+                    if let Some(socket) = client_ip {
+                        let ip = socket.ip();
+                        info!("{ip:?}");
```

Comment:
We already have a line that prints the username on successful login: `info!("{} signed in", username);`. Instead of adding a separate message, it would be better to include the IP address information in this existing line, e.g., `info!("{} signed in from IP: {:?}", username, ip);`. This keeps the logging concise and informative.